rvv-intrinsic-generator/rvv_intrinsic_gen/vector_crypto_inst.py

```diff
@@ -20,7 +20,8 @@ def gen(g):
       UITYPE,
       SEWS,
       LMULS,
-      decorators.has_masking_maskedoff_policy)
+      decorators.has_masking_maskedoff_policy,
+      required_ext_list=["v", "zvkb"])
```

Comment:
https://github.com/riscv/riscv-crypto/blob/main/doc/vector/riscv-crypto-spec-vector.adoc#1-extensions-overview

it should only require zve32x only? and same for most other vector crypto instruction

Reply:
Zvknhb would require SEW=64 (so ELEN >= 64) so I think it would imply more restrictions

Reply:
I am going to add the `zve*` requirement in the runtime, which will make it easier to handle different SEW variation cases.

```
if int(args["SEW"] / lmul_num) == 64:
    inst_info.add_required_ext("zve64x")
else:
    inst_info.add_required_ext("zve32x")
```